.github/boring-cyborg.yml

```
@@ -272,6 +272,15 @@ labelPRBasedOnFilePath:
     - tests/providers/**/secrets/*
     - docs/apache-airflow/administration-and-deployment/security/secrets/**/*
 
+  area:Triggerer:
+    - airflow/cli/commands/triggerer_command.py
+    - airflow/jobs/triggerer_job_runner.py
+    - airflow/models/trigger.py
```

```suggestion
    - airflow/models/trigger.py
    - airflow/triggers/**/*
```
Should we add `airflow/triggers/**/*` too?

---

hmm, I'm not sure about including all triggers from path `airflow/triggers/**/*` but I think `airflow/triggers/base.py` may make sense